workspace/utilities/com/schema-search.xsl

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+
+	<xsl:template name="schema-search">
+		<xsl:param name="url" select="'search-page-url'" />
```

Comment:
we should add a todo word in the fake value and avoid missing to change it. (will show up in the checklist)